js/script.js

```diff
@@ -74,3 +80,25 @@ function addBook(e) {
 }
 
 addBtn.addEventListener('click', addBook);
+
+const navAnchs = document.querySelectorAll('nav ul li a');
+
+navAnchs.forEach((a) => {
+  function navFunction(e) {
+    e.preventDefault();
+    const sections = document.querySelectorAll('section');
+    const linkTarget = e.target.href;
+    sections.forEach((section) => {
+      if (linkTarget.includes(section.id)) {
+        section.classList.remove('d-none');
+      } else {
+        section.classList.add('d-none');
+      }
+    });
+  }
+  a.addEventListener('click', navFunction);
+});
```

Comment:
You did well using [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` methods 😎